apps/admin/frontend/src/components/export_batch_tally_results_button.tsx

```diff
@@ -6,15 +6,22 @@ import { assert } from '@votingworks/basics';
 import { SaveFileToUsb, FileType } from './save_file_to_usb';
 import { AppContext } from '../contexts/app_context';
 import { generateBatchTallyResultsCsv } from '../utils/generate_batch_tally_results_csv';
-import { useCvrFileModeQuery } from '../hooks/use_cvr_file_mode_query';
+import { getCastVoteRecordFileMode } from '../api';
 
-export function ExportBatchTallyResultsButton(): JSX.Element {
+export function ExportBatchTallyResultsButton(): JSX.Element | null {
   const [isSaveModalOpen, setIsSaveModalOpen] = useState(false);
   const { fullElectionTally, electionDefinition } = useContext(AppContext);
   assert(electionDefinition);
   const { election } = electionDefinition;
 
-  const isTestMode = useCvrFileModeQuery().data === Admin.CvrFileMode.Test;
+  const castVoteRecordFileModeQuery = getCastVoteRecordFileMode.useQuery();
+
+  if (!castVoteRecordFileModeQuery.isSuccess) {
+    return null;
+  }
```

Comment:
Returning `null` on this button, I'm not as sure about.

Reply:
OK, will look into options here.

Reply:
this is below the fold so it practically doesn't matter, but that could change in future. for now I switched it to just have the button disabled until the query is ready, which seems to me like a fine pattern